app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/choose/ChooseInsuranceToTerminateDestination.kt

```diff
@@ -200,6 +172,21 @@ private fun ChooseInsuranceToTerminateScreen(
   }
 }
 
+private fun List<TerminatableInsurance>.toRadioOptionDataList(selectedInsuranceId: String?): List<OptionData> {
+  val result = mutableListOf<OptionData>()
+  for (i in this) {
+    result.add(
+      OptionData(
+        id = i.id,
+        optionText = i.displayName,
+        chosenState = if (selectedInsuranceId == i.id) Chosen else NotChosen,
+        labelText = i.contractExposure,
+      ),
+    )
+  }
+  return result
+}
```

Comment:
```suggestion
private fun List<TerminatableInsurance>.toRadioOptionDataList(selectedInsuranceId: String?): List<OptionData> {
  return this.map { i ->
    OptionData(
      id = i.id,
      optionText = i.displayName,
      chosenState = if (selectedInsuranceId == i.id) Chosen else NotChosen,
      labelText = i.contractExposure,
    )
  }
}
```

I am a `mutableList` hater, I must admit 😅